app/src/main/java/com/keylesspalace/tusky/fragment/NotificationsFragment.java

```diff
@@ -646,6 +643,8 @@ private void showFilterMenu() {
         }
         window.setContentView(view);
         window.setFocusable(true);
+        window.setWidth(ViewGroup.LayoutParams.WRAP_CONTENT);
+        window.setHeight(ViewGroup.LayoutParams.WRAP_CONTENT);
```

Comment:
this fixes a bug on Android 5 where the `PopupWindow` just wasn't shown. See http://eng.moldedbits.com/technical/android/2016/09/29/android-popup-window.html